src/client/cli/cmd/shell.cpp

```diff
@@ -117,12 +117,12 @@ std::vector<std::string> cmd::Shell::aliases() const
 
 QString cmd::Shell::short_help() const
 {
-    return QStringLiteral("Open a shell on a running instance");
+    return QStringLiteral("Open a shell on the primary or specified instance");
```

Comment:
I'd simplify the short help further by just saying: 
```suggestion
    return QStringLiteral("Open a shell on an instance");
```